packages/react/src/primitives/shared/responsive/utils.ts

```diff
@@ -3,10 +3,10 @@ import { isDesignToken } from '@aws-amplify/ui';
 import { Breakpoint, Breakpoints } from '../../types/responsive';
 
 export const getValueAtCurrentBreakpoint = (
-  values: Record<string, any> | string[] | string,
+  values: Record<string, string> | string[] | string,
```

Comment:
Had to add return types to `getValueAtCurrentBreakpoint` and `getClosestValueByBreakpoint` in order to allow `useBreakpointValue` to pass linting. Additionally, updated `any` to `string` based on the usage throughout the code base